shared_libs/transformer_base_lib/transformer_base_lib/run_transformer.py

```diff
@@ -22,24 +22,51 @@
 
 
 def run_transformer_app(transformer: BaseTransformer) -> None:
-    
     from .base_transformer import BaseTransformer
-    
+
     if not isinstance(transformer, BaseTransformer):
         raise TypeError("transformer must be an instance of BaseTransformer")
-    
+
     processor_manager = ProcessorManager()
-    config = TransformerConfig.from_env_and_config_file(transformer.config_path)
-    
-    client_config = ConnectionConfig(config.connection_string, config.service_bus_namespace)
+
+    config_path = transformer.config_path
+    if config_path is not None:
+        logger.debug(
```

Comment:
this logs should go into the config factory method

Reply:
done